src/nvim/eval.c

```diff
@@ -10386,6 +10386,12 @@ void script_host_eval(char *name, typval_T *argvars, typval_T *rettv)
   *rettv = eval_call_provider(name, "eval", args);
 }
 
+void eval_call_provider_discard(char *provider, char *method, list_T *arguments)
+{
+  typval_T rettv = eval_call_provider(provider, method, arguments);
+  tv_clear(&rettv);
+}
+
 typval_T eval_call_provider(char *provider, char *method, list_T *arguments)
```

Comment:
can we do it as a param instead?
```suggestion
/// @param discard  Clears the value returned by the provider and returns empty typval_T. 
typval_T eval_call_provider(char *provider, char *method, list_T *arguments, bool discard)
```

Reply:
Done.